### ISSUE TITLE
Tab order in modal controls can't be reversed

### ISSUE THREAD

**Issue by cmachanic** [biati-digital/glightbox#356]
**Describe the bug**
When tabbing between the previous, next and close buttons, the tab sequence cannot be reversed with **Shift + Tab**. It always moves clockwise.

**Are you able to reproduce the bug in the demo site**
Yes

**To Reproduce**
1. Open modal and tab between the previous, next and close buttons.
2. Try to reverse the tabbing sequence (counter clockwise) with the key command Shift + Tab, and the tabbing continues in the same sequence.

**Expected behavior**
A keyboard user should be able to go back and forth between the previous and next buttons

**Desktop:**
 - Tested on MacOS Chrome, Firefox
 - Tested on Windows Edge, Chrome and Firefox

**Comment by suanTech** [comment]
Hi @biati-digital any updates on this? The reverse tab still doesn't work. And when I try to override the default tab behaviour it "jumps around" as mentioned in the other issue.
